radix-engine/src/system/system_modules/costing/fee_table.rs

```diff
@@ -176,10 +176,11 @@ impl FeeTable {
         _export_name: &str,
         wasm_execution_units: u32,
     ) -> u32 {
-        // From `costing::spin_loop`, it takes 5.5391 ms for 1918122691 wasm execution units.
-        // Therefore, cost for single unit: 5.5391 *  1000 / 1918122691 * 100 = 0.00028877714
+        // From `costing::spin_loop`, it takes 10.824 ms for 2918134342 wasm execution units
+        // (measured at Apple M1 Max, git rev 680d511cdb)
+        // Therefore, cost for single unit: 10.824 *  1000 / 2918134342 * 100 = 0.0003709 ~= 0.0004
 
-        wasm_execution_units / 3000
```

Comment:
Something here doesn't seem right -
1 / 0.00028877714 = 3462 which rounded down to 3000
1 / 0.0003709 = 2696 which would round down to 2500, not 4000

That said, using 2500 would make wasm more expensive. Maybe a few more of us should run this to provide a larger sample size?

Reply:
I am inventing new maths to reduce WASM costs 😁 
Agree, this calculation is wrong.

However, this `5.5391 ms` duration was wrong too. Current durations at 
- Apple M1 Max is `10.8ms` which gives
`1 / 0.000563 = 1776`
- GH CI `17.8 ms` which gives
`1 / 0.000928 = 1077`

So we were undercharging for WASM.

I suggest we use CI benchmark times for calculations. Which would give us
`17.8 * 1000 / 2918134342 * 100 = 0.000609978; 1 / 0.000609978 = 1639`

Reply:
As discussed, this legacy `spin_loop` has been using `NoOpWasmRuntime`, which has a different performance profile than the standard runtime. 

We've updated the benchmarks as part of #1982 - no undercharging is observed except for the `spin_loop_v2` case.